CONTRIBUTING.md

```diff
@@ -23,6 +23,105 @@ First of all, thank you for contributing. It’s appreciated.
 
 ## Component structure
 
+### Create a new Component
+Here are some basic steps to get you started building your first component.
+
+1. Create your new component folder
+`src/components/MyNewComponent/`
+2. Create your component file(s)
+```
+    /MyNewComponent/
+      |-examples/
+        |-0.default.jsx <-- default example, used by `gulp docs-generate` to create component demo
+      |-MyNewComponent.jsx  <-- component logic goes here
+      |-MyNewComponent/MyNewComponent.less <-- component styling goes here
+      |-MyNewComponent/MyNewComponent.spec.js <-- component tests go here
+  ```
+3. Component Template
+```javascript
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { lucidClassNames } from '../../util/style-helpers';
+import { createClass, findTypes, omitProps } from '../../util/component-types';
+
+const cx = lucidClassNames.bind('&-MyNewComponent');
+
+const { string, bool, node } = PropTypes;
+
+const MyNewComponent = createClass({
+	displayName: 'MyNewComponent',
+	propTypes: {
```

Comment:
having a few additional prop types would be useful for the purposes of an example
like some handler functions, or some boolean or numeric prop types
just so people can see a more fleshed out example